### PR TITLE
[VEGA-5306] change "about project" layout

### PR DESCRIPTION
**Pull request by Alexandr1111**
## Задача
Задача в Jira GPN: https://jira.konakov.tv/browse/VEGA-5306

## Задача
https://artcpt.atlassian.net/browse/VEGA-5306
## Problem statement
В связи с появлением КЛ и ТЭОС изменена разметка страницы проекта

## Solution details

_{What has been done (list of details)}_

## Type
- [x] Feature
- [ ] Bug Fix
- [ ] Refactoring
- [ ] Test
- [ ] Other

## Associated issues

_{Jira issues or PRs related to this one}_

## Quality control
- [x] PR: Based on a correct branch
- [x] PR: Head branch is rebased on the base branch
- [x] PR: Assignee chosen and necessary labels are set
- [x] PR: Current form is filled out (where it makes sense)
- [x] PR: Diff checked for irrelevant changes
- [x] PR: Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specs
- [x] PR: Checked for spelling, grammar and typos
- [x] JS: There's no errors/warnings in the browser dev console
- [x] Layout: Tested with various content amounts
- [ ] Docs: All the important changes and features are described
- [ ] Tests: New unit tests developed
- [ ] Tests: New E2E tests developed
- [x] Tests: Existing unit tests passed successfully
- [x] Tests: Existing E2E tests passed successfully
- [x] Tests: Manually tested on personal instance

## Notable statements
- [ ] Affects configuration files
- [ ] Brings new packages or updates existing
- [ ] Opened follow-up tasks to resolve

e2e(падение одного нестабильного):
http://jenkins-remote-ci-gpn-vega-builder.vega2.org:8081/job/STAND-TOOLBOX/job/scenario-tests-on-demand/2238/

### CODE REVIEW

File: src/ui/features/projects/ProjectForm/ProjectForm.tsx [hunk 1]
```diff
@@ -147,36 +152,31 @@ export const ProjectForm: React.FC<FormProps> = (formProps) => {
...
+              data-testid={testId.stepList}
+            >
+              <ModuleNavigation currentProjectId={currentProjectId} />
             </div>
+            {(currentProjectId || mode === 'create') && (
+              <div
+                className={cnProjectForm('Content')}
+                style={{
+                  background: '#22272b',
+                  gridArea: 'activeTabContent',
```
Comment (SergeyKatugin) [inline src/ui/features/projects/ProjectForm/ProjectForm.tsx:167]: стиль остался

File: src/ui/features/projects/ProjectForm/moduleNavigation/ModuleNavigation.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+}) => {
+  const history = useHistory();
+
+  return (
+    <button
+      type="button"
+      key={title}
+      onClick={() => {
+        if (url) {
+          history.push(url);
```
Comment (SergeyKatugin) [inline src/ui/features/projects/ProjectForm/moduleNavigation/ModuleNavigation.tsx:28]: условия нужно выносить

File: src/ui/features/projects/ProjectForm/tabs/Tabs.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      }).toString();
+    };
+
+    return (
+      <div className={cnProjectFormTabs('Tab')}>
+        <Text
+          size="s"
+          as="button"
+          role="tab"
+          onClick={!disabled ? onChange : () => {}}
```
Comment (SergeyKatugin) [inline src/ui/features/projects/ProjectForm/tabs/Tabs.tsx:65]: такие штуки надо выносить

File: src/ui/features/projects/ProjectForm/tabs/Tabs.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+          data-testid={testId}
+          font="primary"
+          view={disabled ? 'ghost' : 'primary'}
```
Comment (SergeyKatugin) [inline src/ui/features/projects/ProjectForm/tabs/Tabs.tsx:68]: и это, в memo

File: src/ui/features/projects/ProjectForm/tabs/Tabs.tsx (hunk 3 above)
Comment (Alexandr1111) [reply src/ui/features/projects/ProjectForm/tabs/Tabs.tsx:65]: Готово

File: src/ui/features/projects/ProjectForm/moduleNavigation/ModuleNavigation.tsx (hunk 2 above)
Comment (Alexandr1111) [reply src/ui/features/projects/ProjectForm/moduleNavigation/ModuleNavigation.tsx:28]: Готово

File: src/ui/features/projects/ProjectForm/tabs/Tabs.tsx (hunk 4 above)
Comment (Alexandr1111) [reply src/ui/features/projects/ProjectForm/tabs/Tabs.tsx:72]: Оно и так всё в мемо, поэтому даже не знаю..

Comment (SergeyKatugin) [reply src/ui/features/projects/ProjectForm/tabs/Tabs.tsx:72]: Ок

**Pull request merged**